driver/interrupts.h

```diff
@@ -1,8 +1,15 @@
 /* Interface to interrupt handling. */
 
+#define N_EVENT_READERS (8)
```

Comment:
Brackets on a plain constant?  Seems overly pedantic to me.

Reply:
Fixed